DQM/HLTEvF/python/HLTObjectsMonitor_B2G_cfi.py

```diff
@@ -0,0 +1,83 @@
+import FWCore.ParameterSet.Config as cms
+
+b2gObjects = cms.VPSet(
+       cms.PSet(
```

Comment:
wow, they are a lot... I agree it would be nicer to have a default somewhere (maybe with all the optional parts set to `False`) and just switch on the parts needed case by case.